kafka-observ-lib/signals/topic.libsonnet

```diff
@@ -9,7 +9,7 @@ function(this)
     aggFunction: 'sum',
     legendCustomTemplate: '{{ topic }}',
     discoveryMetric: {
-      prometheus: 'kafka_topic_partition_current_offset',  //https://github.com/danielqsj/kafka_exporter?tab=readme-ov-file#metrics
+      prometheus: 'kafka_log_log_logstartoffset',  //https://github.com/danielqsj/kafka_exporter?tab=readme-ov-file#metrics
```

Comment:
Just to doublecheck, the metric is `kafka_log_log_logstartoffset`, with two `log`s in there?

Reply:
yepp, it is an actual name.